circle.yml

```diff
@@ -0,0 +1,82 @@
+version: 2
+workflows:
+  version: 2
+  linux-builds:
+    jobs:
+      - build-x86_64
+      - build-i686
+
+jobs:
+  build-x86_64:
+    docker:
+#      - image: ubuntu:14.04
+      - image: circleci/python:2.7
+        environment:
+          JULIA_CPU_CORES: 4
+          JULIA_TEST_MAXRSS_MB: 800
+          ARCH: x86_64
+#    resource_class: xlarge
+    steps: &steps
+#          apt-get update &&
+#            curl make patch m4 xz-utils git time ccache bar
+      - run: | # install build dependencies
```

Comment:
sure but I personally would rather see the command that's running as the name of the step, the comments are more explanation for anyone who might need to modify the file